great_tables/_export.py

```diff
@@ -239,17 +222,25 @@ def save(
     debug_port
         Port number to use for debugging. By default no debugging port is opened.
     encoding
-        The encoding used when writing temporary files.
+        The character encoding used for the HTML content.
     _debug_dump
         Whether the saved image should be a big browser window, with key elements outlined. This is
         helpful for debugging this function's resizing, cropping heuristics. This is an internal
         parameter and subject to change.
+    **params
```

Comment:
Is it okay if we remove the `**params` piece of this PR? I'm onboard with everything else. Because `**params` only applies to when we go from a `.png` -> something else, it exposes PIL as part of the user API.

If we leave it out, we can swap out PIL down the road. As an alternative could we tell users that we're using Pillow? We could even tell them about PIL open and save if needed (so they can go from .png -> anything PIL supports on their own if more customization is needed).

Reply:
Thanks for the explanation. I agree that we should remove `**params`, making `Pillow` an internal dependency rather than a public one. 

Regarding hints, I believe that diligent readers likely already recognize we use `Pillow` [under the hood](https://posit-dev.github.io/great-tables/reference/GT.save.html#details) and would consult its documentation for further customization if needed. 

This aligns with our goal of providing a user-friendly way to save generated tables rather than focusing on creating highly customized table figures in various formats🤔.